packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -0,0 +1,160 @@
+
+### Step 1: First, we'll have to create our own telemetry consumer which extends the ITelemetryConsumer interface using our Azure App Insights client:
+
+```ts
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { ITelemetryConsumer } from "@fluidframework/fluid-telemetry";
+
+class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
+	constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
+	consume(event: IFluidTelemetry) {
+		this.appInsightsClient.trackEvent({
+			name: event.eventName,
+			properties: event,
+		});
+	}
+}
+```
+
+#### Step 2: Now, let's start the telemetry production and hook in our telemetry consumer from step 1. We will be initializing our telemetry collection where we initialize our containers:
+
+```ts
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { IFluidContainer } from "@fluidframework/fluid-static";
+import { ITelemetryConsumer , TelemetryConfig, startTelemetry, IFluidTelemetry } from "@fluidframework/external-telemetry"
+
+// 1: This is supposed to be your code for loading a Fluid Container
+let myAppContainer: IFluidContainer;
+let myAppContainerId: string;
+if (containerExists) {
+    myAppContainerId = {...your code to get the id of the existing container}
+    myAppContainer = {...your code to load a Fluid Container from myAppContainerId}
+} else {
+    myAppContainer = {...your code to create a new Fluid Container}
+    myAppContainerId = await myAppContainer.attach();
+}
+
+// 2: This is our implementation of ITelemetryConsumer that will send telemetry to Azure App Insights
+class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
+    constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
+    consume(event: IFluidTelemetry) {
+        this.appInsightsClient.trackEvent({
+            name: event.eventName,
+            properties: event,
+        });
+    }
+}
+
+// 3. Next, we'll Create the telemetry config object.
+// Note that we have to obtain the containerId before we can do this.
+const telemetryConfig: TelemetryConfig = {
+    container: myAppContainer,
+    containerId: myAppContainerId,
+    consumers: [new AppInsightsTelemetryConsumer(appInsightsClient)],
+};
+
+// 4. Start Telemetry
+startTelemetry(telemetryConfig);
+
+// Done! Your container telemetry is now being created and sent to your Telemetry Consumer which will forward it to Azure App Insights.
+```
+
+Congrats, that's it for now! If you've decided to use Azure App Insights, we have designed useful prebuilt queries for you that utilize the generated telemetry
+
+# Telemetry Events
+
+This section details the currently available telemetry event and their typescript types.
+
+### Container Telemetry
+
+Telemetry events relating directly to Fluid Containers.
+
+1. `ContainerConnectedTelemetry`
+1. `ContainerDisconnectedTelemetry`
+1. `ContainerDisposedTelemetry`
```

Comment:
I'd probably also remove this. Without further details it's not particularly useful, and we can add those as a follow up task.

Reply:
Removed.